docs/Development.md

```diff
@@ -94,7 +94,7 @@ and you will just have to run `git commit -a` again.
 
 ## Testing ##
 
-To run tests, `cd` into the repository and run `python setup.py test` or `pytest`.
+To run tests, `cd` into the repository and run `python -m pytest`.
```

Comment:
Maybe use this opportunity to add a mention of `pip install -r test/requirements.txt`?

Also i'm used to simply calling `pytest`, but I guess `python -m pytest` is exactly the same.

Reply:
Thanks, I'll mention it.

And there's a difference between `pytest` and `python -m pytest`: https://github.com/jugmac00/til/blob/master/python/difference-between-pytest-and-python-m-pytest.md but it applies only of the project is under `src/`. In this case `pytest` is enough. I'll change it.